src/client/src/components/pages/dashboard/index.jsx

```diff
@@ -11,7 +12,9 @@ class index extends Component {
   state = {
     participant: "",
     course: "",
-    worker: ""
+    worker: "",
+    employed: "",
+    notEmployed: ""
```

Comment:
Since employed and notEmployed will be numbers, it might make this code more predictable if they are numbers (I suppose 0) in initialState too.

Reply:
@AsalaKM 
I looked. All the changes look good together! Thanks for doing them.

Reply:
Thanks for you :star2: